INSTALL.md

```diff
@@ -214,24 +230,24 @@ image. We'll use
 ### Linux
 
 ```sh
-.../cryptol-course> cryptol labs/Demos/OneTimePad.md
+.../cryptol-course$ cryptol labs/Demos/OneTimePad.md
```

Comment:
Good catch!

Reply:
Most recent push removed redundancy in this section.